src/cmd-buildextend-hashlist-experimental

```diff
@@ -94,7 +94,7 @@ class HashListV1(dict):
         # Force all dirs to be owner writable so that we can delete the
         # checkout at the end. XXX: Lower into a new `ostree checkout` flag.
```

Comment:
Agree. It would be nice to include some of the context in https://github.com/coreos/fedora-coreos-tracker/issues/1429#issuecomment-1453701635 here and in the commit message.

Reply:
got it, I have updated the comments and changed the commit message.